src/lib/helper.js

```diff
@@ -32,6 +32,16 @@ const fromMilliSecToHours = (time) => time / (1000 * 60 * 60);
 
 const fromMilliSecToDays = (time) => Math.ceil(time / (1000 * 60 * 60 * 24));
```

Comment:
Sorry I wanted to add this comment on last week issues :) What do you think about giving more semantics to this? **( 1000 * 60 * 60 * 24 )**? We usually call this Magic numbers, https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
Didn't know about magic numbers, so thank you for adding the link. And I think giving it more semantics sounds great. :))